CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v0.21.0 (2023-02-21)
+
+### ✨ New
+
+- [#1410](https://github.com/meltano/sdk/issues/1410) Build empty cookiecutters and run lint task during CI -- _**Thanks @flexponsive!**_
+
+### 🐛 Fixes
+
+- [#1428](https://github.com/meltano/sdk/issues/1428) E2E Cookiecutter - Cover all REST authentication cases + one GraphQL case -- _**Thanks @flexponsive!**_
```

Comment:
@edgarrmondragon Could you please update these change messages to something more clear